zstacklib/zstacklib/utils/iscsi.py

```
@@ -0,0 +1,187 @@
+        matches = disk_regex.search(line)
+        if not matches:
+            continue
+
+        disk_path = "/dev/%s" % matches.group(1)
+        if get_iscsi_device_serial(disk_path) == id:
+            return disk_path
+
+    return ""
+
+
+class IscsiLogin(object):
```

Pylint (3.3.7)

[refactor] Class 'IscsiLogin' inherits from object, can be safely removed from bases in python3

(R0205)